### PR TITLE
Update tflite-converter-issue template

### PR DESCRIPTION
**Pull request by ymodak**
Updating TF Lite Converter Issue template to simplify issues debugging.

### CODE REVIEW

**Review by abattery — COMMENTED** [review]
It would be nice to give a sense of what TensorFlow model is. 

... your TensorFlow model, for example, Saved Model, Keras or Graph def, and ...

File: .github/ISSUE_TEMPLATE/60-tflite-converter-issue.md [hunk 1]
```diff
@@ -5,42 +5,50 @@ labels: 'TFLiteConverter'
...
-# Copy and paste the output here.
+# Provide your code here (or put links here or attach to the issue)
+# - Include code to invoke the TFLite Converter Python API
+# - Include the output (with errors) from the converter invocation 
 ```
 
-**Also, please include a link to the saved model or GraphDef**
+### 3. Models
+
+Provide links to your TensorFlow model and (optionally) TensorFlow Lite Model
```
Comment (abattery) [inline .github/ISSUE_TEMPLATE/60-tflite-converter-issue.md:36]: It would be nice to give a sense of what TensorFlow model is.

... your TensorFlow model, for example, Saved Model, Keras or Graph def, and ...

File: .github/ISSUE_TEMPLATE/60-tflite-converter-issue.md [hunk 2]
```diff
@@ -5,42 +5,50 @@ labels: 'TFLiteConverter'
...
 
+### 2. Code (that demonstrates how to reproduce your issue)
+**Option A: Using reference colab notebooks**
 
-**Command used to run the converter or code if you’re using the Python API**
-If possible, please share a link to Colab/Jupyter/any notebook.
+The reference colab notebooks given below demonstrate: *(TensorFlow Model Colab)* Build model in TF Keras --> *(TensorFlow Lite Model Colab)* Convert to TF Lite (performing quantization techniques) and run TFLite Inference. You may use these colab notebooks as a reference point to generate your model behavior and attach links.
+
+1)  Reference [TensorFlow Model Colab](https://colab.research.google.com/gist/ymodak/e96a4270b953201d5362c61c1e8b78aa/tensorflow-datasets.ipynb?authuser=1)
+2)  Reference [TensorFlow Lite Model Colab](https://colab.research.google.com/gist/ymodak/0dfeb28255e189c5c48d9093f296e9a8/tensorflow-lite-debugger-colab.ipynb)
```
Comment (abattery) [inline .github/ISSUE_TEMPLATE/60-tflite-converter-issue.md:21]: Those are nice colab links. How about keeping them somewhere in TensorFlow Lite repository?

**Comment by gbaned** [comment]
@ymodak Can you please resolve conflicts? Thanks!

**Comment by abattery** [comment]
Already merged the PR internally.
